src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -314,22 +317,25 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
 
     val stderrFileLength = Try(Files.size(stderr)).getOrElse(0L)
     val failOnErrFlag = failOnStderr.getOrElse("false").toBoolean
-    val rc = returnCode.contentAsString.stripLineEnd.toInt
+    val rc = Try(returnCode.contentAsString.stripLineEnd.toInt)
```

Comment:
Not a big deal on the local backend, but this could be made lazy so none of this work is done if the `processReturnCode` is nonzero.  This is a bigger deal on non-local backends where that file would need to be downloaded but here it's just a nicety.  :smile: